shared/common-adapters/button.desktop.js

```diff
@@ -13,6 +13,9 @@ class Button extends Component {
     let backgroundStyle = {}
     let labelStyle = {}
     let progressColor = globalColors.white
+    let rippleStyle: {rippleColor: string} = {}
+    // Set all button ripples to be more prominent
+    rippleStyle = {rippleColor: 'rgba(0, 0, 0, 0.3)'}
```

Comment:
I wouldn't as the ripple and all the material ui stuff isn't really a part
of the design. I think this is an improvement over the existing behavior
but I wouldn't be surprised if the ripple disappears entirely

On Friday, June 3, 2016, Max Goodman <notifications@github.com> wrote:

> In shared/common-adapters/button.desktop.js
> <https://github.com/keybase/client/pull/2986#discussion_r65785967>:
>
> > @@ -13,6 +13,9 @@ class Button extends Component {
> >      let backgroundStyle = {}
> >      let labelStyle = {}
> >      let progressColor = globalColors.white
> > +    let rippleStyle: {rippleColor: string} = {}
> > +    // Set all button ripples to be more prominent
> > +    rippleStyle = {rippleColor: 'rgba(0, 0, 0, 0.3)'}
>
> @chrisnojima <https://github.com/chrisnojima> Do you think this should be
> added to / sourced from style-guide-colors.js?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/2986/files/b32a0ed3e0cfbc9354f0d4b7f3c75221185cd444#r65785967>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAfhU6fS2u5vth3rQ3CVgFYR_NivRmcoks5qIK1UgaJpZM4It685>
> .
>